lib/otw_sanitize/embed_sanitizer.rb

```diff
@@ -161,7 +190,7 @@ class type width
         disable_scripts(node)
         node['flashvars'] = "" unless allows_flashvars?
       end
-      { node_whitelist: [node, parent] }
+      { node_whitelist: [node] }
```

Comment:
Could you explain this change? Or rather, why it was necessary as part of this fix? (It's fine if it wasn't; I'm just utterly lost.)

Reply:
https://github.com/otwcode/otwarchive/blob/0622729c9d4fca1ec7b611235bb5e318256ebd70/lib/otw_sanitize/embed_sanitizer.rb#L74-L79

This transformer looks for either:

- `<param>` tags with `<object>` parents, in which case we sanitize and safelist both the current node and the parent node.
- `<embed>`/`<iframe>` tags, in which case we sanitize just the current node and don't look at the parent. So we shouldn't safelist the parent in these cases.